backend/functions/src/api/resources/user-profile/user-profile.controller.ts

```diff
@@ -0,0 +1,22 @@
+import { RequestHandler } from 'express';
+import { UserProfile } from '.';
+import { getUserProfileService, updateUserProfileService } from './user-profile.service';
+
+export const getUserProfile: RequestHandler = async (req, res) => {
+  try {
+    const profile = await getUserProfileService(req.params.id);
+    res.send(profile);
+  } catch (error) {
+    res.send(error);
+  }
+};
+
+export const patchUserProfile: RequestHandler = async (req, res) => {
+  try {
+    const userProfile: UserProfile = req.body;
+    const profile = await updateUserProfileService(req.params.id, userProfile);
+    res.send(profile);
+  } catch (error) {
+    console.log(error);
```

Comment:
You should always return a response when a request is received. If you do not, the consumer of this API (in this case the frontend app) will never know what happened, and will keep waiting unnecessarily for this request.

Also, I believe we have an universal error handler for the Express app, no? So why are you manually handling errors?